### PR TITLE
[bitnami/kubewatch] Add priorityClassName support

### PR DESCRIPTION
**Pull request by bugok**
**Description of the change**

This adds the ability to set the `priorityClassName` property to the kubewatch pods.
This is copied from the memcached chart.

**Benefits**
This will enable to give a higher priority to the kubewatch pods.

**Possible drawbacks**
Can't think about any - as it's backwards compatible.

**Additional information**

Testing was done like so:
```
helm dependency build kubewatch
helm package kubewatch
helm upgrade -i kubewatch ~/sources/charts-1/bitnami/kubewatch-3.1.4.tgz -f ~/tmp/kubewatch.yaml -n monitoring
```

which contains this:
```
priorityClassName: 'system-cluster-critical'
```

See that the change took effect:
```
$ kubectl get pods -n monitoring | grep kubewatch
kubewatch-758c5b9745-4rwl8                                        1/1     Running   0          8m34s
```

See that the spec matches:
```
$ kubectl get pod kubewatch-758c5b9745-4rwl8 -n monitoring -o template --template={{.spec.priorityClassName}}'
system-cluster-critical
kubectl get pod kubewatch-758c5b9745-4rwl8 -n monitoring -o template --template={{.spec.priority}}
2000000000
```

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/kubewatch/Chart.yaml [hunk 1]
```diff
@@ -28,4 +28,4 @@ name: kubewatch
 sources:
   - https://github.com/bitnami/bitnami-docker-kubewatch
   - https://github.com/bitnami-labs/kubewatch
-version: 3.1.3
+version: 3.1.4
```
Comment (javsalgar) [inline bitnami/kubewatch/Chart.yaml:31]: As it is adding a non-breaking feature, could you bump the minor version?

**Review by javsalgar — CHANGES_REQUESTED** [review]
Thanks for the PR! Small change

**Comment by bugok** [comment]
I might have messed my branch here. Although the changed files look OK.

**Review by javsalgar — APPROVED** [review]
LGTM! Thanks

**Pull request merged**